tests/unit/sagemaker/serve/utils/test_task.py

```diff
@@ -18,7 +18,7 @@
 
 from sagemaker.serve.utils import task
 
-EXPECTED_INPUTS = {"inputs": "Paris is the <mask> of France.", "parameters": {}}
+EXPECTED_INPUTS = {"inputs": "Paris is the [MASK] of France.", "parameters": {}}
```

Comment:
Pull latest commit from master, it should cover this case

Reply:
This would not have any impact when merged if master already has it, wouldn't it?

Reply:
No I think you would need to pull, check PR checks

This branch has conflicts that must be resolved
Only those with [write access](https://docs.github.com/articles/what-are-the-different-access-permissions) to this repository can merge pull requests.